### PR TITLE
Use safe ares in storyboards.

### LINKED ISSUE #302: Use Safe areas.
see https://useyourloaf.com/blog/safe-area-layout-guide/

+ revise storyboards

### PR DESCRIPTION
**Pull request by mathebox**
Fixes #302

### CODE REVIEW

File: xikolo-ios/Layouts/Base.lproj/TabDashboard.storyboard [hunk 1]
```diff
@@ -130,7 +130,7 @@
         <scene sceneID="2oD-bA-n2d">
             <objects>
                 <collectionViewController id="siC-S8-0pE" customClass="CourseActivityViewController" customModule="moocHOUSE" customModuleProvider="target" sceneMemberID="viewController">
-                    <collectionView key="view" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" dataMode="prototypes" id="sph-AR-Bam">
+                    <collectionView key="view" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" misplaced="YES" dataMode="prototypes" id="sph-AR-Bam">
```
Comment (bjrne) [inline xikolo-ios/Layouts/Base.lproj/TabDashboard.storyboard]: Sure?

File: xikolo-ios/Layouts/Base.lproj/TabNews.storyboard [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="12089"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="13173"/>
         <capability name="Constraints to layout margins" minToolsVersion="6.0"/>
         <capability name="Constraints with non-1.0 multipliers" minToolsVersion="5.1"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
+    <customFonts key="customFonts">
+        <array key="HelveticaNeue.ttc">
+            <string>HelveticaNeue</string>
```
Comment (bjrne) [inline xikolo-ios/Layouts/Base.lproj/TabNews.storyboard]: Do we actually use it? Or was that a fault in any textview?

File: xikolo-ios/Layouts/Base.lproj/TabDashboard.storyboard (hunk 1 above)
Comment (mathebox) [reply xikolo-ios/Layouts/Base.lproj/TabDashboard.storyboard]: Fixes

File: xikolo-ios/Layouts/Base.lproj/TabNews.storyboard (hunk 2 above)
Comment (mathebox) [reply xikolo-ios/Layouts/Base.lproj/TabNews.storyboard]: Weird.. the announcement text was not formatted with the system font.

Comment (mathebox) [reply xikolo-ios/Layouts/Base.lproj/TabNews.storyboard]: Thanks for your extensive review!

**Pull request merged**